src/charts/bar.tsx

```diff
@@ -112,15 +112,7 @@ export const semioticBarChart = (
     };
   }
 
-  const uniqueValues = sortedData.reduce(
-    (uniques, datapoint) =>
-      !uniques.find(
-        (uniqueDimName: string) => uniqueDimName === datapoint[dim1].toString(),
-      )
-        ? [...uniques, datapoint[dim1].toString()]
-        : uniques,
-    [],
-  );
+  const uniqueValues = dim1 === "none" ? [] : getUniqueValues(sortedData, dim1);
```

Comment:
If a user provides a dataset with  a  column called `none`, it's going to break other things in the `data-explorer` because `none` is already heavily used internally. If we rewrite this in the future, we may want to use a namespaced constant like `$$_DX_NONE_$$` for our `none` column name to reduce the risk of collision with a user's dataframe. For now, I think it's out of scope to change that at the same time.